packages/headless/src/features/triggers/triggers-slice.test.ts

```diff
@@ -1,233 +1,92 @@
 import {buildMockSearch} from '../../test/mock-search';
```

Comment:
This is almost identical to the commerce triggers-slice-tests (https://github.com/coveo/ui-kit/pull/3898/files#diff-4b3628fa3e0f90d193221c704dc72c90b96b3c72e8e46591942d853f29858099) except there are fewer cases to test (because the Commerce API returns triggers in product listing calls as well)